pkg/metrics/metrics.go

```diff
@@ -2,12 +2,18 @@ package metrics
 
 import (
 	"github.com/kyma-incubator/reconciler/pkg/cluster"
+	"github.com/kyma-incubator/reconciler/pkg/scheduler/occupancy"
 	"github.com/prometheus/client_golang/prometheus"
 	"go.uber.org/zap"
 )
 
-func RegisterAll(inventory cluster.Inventory, logger *zap.SugaredLogger) {
+func RegisterAll(inventory cluster.Inventory, occupancyRepo occupancy.Repository, reconcilerList []string, logger *zap.SugaredLogger, occupancyTracking bool) {
 	reconciliationWaitingCollector := NewReconciliationWaitingCollector(inventory, logger)
```

Comment:
Probably better to split that into decicated methods.. one for inventory and another one for the occupancy.